### PR TITLE
updated integration tests reporter

### PR DESCRIPTION
**Pull request by arieznikov**
This will introduce better reporting for integration tests step
https://app.asana.com/0/746943642536769/1130800126316371/f

### CODE REVIEW

**Comment by matt-thomason** [comment]
@arieznikov Does this have a corresponding Asana task? If so, please link it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,11 +59,19 @@ jobs:
               git clone git@github.com:BoltApp/integration-tests.git
               cd integration-tests
               npm install
-              mkdir -p /tmp/screenshots
-              SCREENSHOT_DIR=/tmp/screenshots TEST_ENV=sandbox npm run test-checkout-magento1
+              mkdir -p ./screenshots
```
Comment (matt-thomason) [inline .circleci/config.yml]: Why move the `screenshots` directory? Not that I object, but if we're changing our conventions there should probably be a reason.

File: .circleci/config.yml [hunk 2]
```diff
@@ -59,11 +59,19 @@ jobs:
...
+          path: ./test-results
       - store_artifacts:
-          path: /tmp/screenshots
+          path: ./screenshots
+      - run:
+          name: Generate Report
+          command: npm run generate-report
+          when: always
+      - store_artifacts:
+          path: ./allure-report
```
Comment (matt-thomason) [inline .circleci/config.yml]: Do we need to `mkdir` this directory?

Comment (arieznikov) [reply .circleci/config.yml]: no, it is generated by generate-report

File: .circleci/config.yml (hunk 1 above)
Comment (arieznikov) [reply .circleci/config.yml]: why is it in tmp? ci creates own space for each sub-task there is no need for tmp.

**Comment by mccottry** [comment]
@arieznikov  Make sure to add reviews as subtask in Asana.  Thanks!

**Pull request merged**